### ISSUE TITLE
Side-by-side images with knitr::include_graphics

### ISSUE THREAD

**Issue by mikelove** [Bioconductor/BiocStyle#52]
I'm trying to include some images side by side in a figure, but can't seem to see how to use the [bookdown recommended code](https://bookdown.org/yihui/bookdown/figures.html) with BiocStyle. It becomes rendered as multiple separate figures, e.g. try the following (I had to escape the backticks so you can see them here)

```
---
title: "Test"
author: "Mike"
output: BiocStyle::html_document
---

# Testing side-by-side images

\```{r pacman, echo=FALSE, fig.cap="test"}
knitr::include_graphics(rep("pacman.png",3))
\```

\```{r pacman-solo, echo=FALSE, fig.cap="test"}
knitr::include_graphics("pacman.png")
\```
```

**Comment by aoles** [comment]
Interesting, as swapping `BiocStyle::html_document` to `bookdown::html_document2` doesn't seem to produce side-by-side figures either. If you experience this too, you might consider checking with the bookdown guys what their thoughts on this are.

**Comment by mikelove** [comment]
Good point! I also tried `rmarkdown::html_document` and I do get side-by-side images.

Any pointers on how one should go about putting side-by-side images into a BiocStyle doc with html output, even outside of the figure environment?

With `rmarkdown::html_document`, I can get them with:

```
<img width="30%" src="pacman.png"/>
<img width="30%" src="pacman.png"/>
<img width="30%" src="pacman.png"/>
```

But if I switch to `BiocStyle::html_document` these are no longer side by side.

**Comment by mikelove** [comment]
Here is how I get side by side (w/ rmarkdown 1.9):

````
---
title: "Test"
author: "Mike"
output: rmarkdown::html_document
---

# Testing side-by-side images:

<img width="30%" src="pacman.png"/>
<img width="30%" src="pacman.png"/>
<img width="30%" src="pacman.png"/>
````

![screen shot 2018-06-25 at 10 33 41 am](https://user-images.githubusercontent.com/69203/41856509-4698aa9c-7863-11e8-8299-eb374ca7efb5.png)

**Comment by aoles** [comment]
This approach won't work with BiocStyle because of the way the CSS is defined for positioning the `<img>` elements. You can override this through the following customization. I will need to investigate this further, though, in order to decide whether a general fix is possible.
```
---
output:
  BiocStyle::html_document:
    css: custom.css
---
```
**custom.css**
```css
img {
  display: inline;
}
```

**Comment by mikelove** [comment]
Thanks @aoles no rush from my side
